### ISSUE TITLE
Feature Request: Add option to print the enviroment with the command

### ISSUE THREAD

**Issue by elichai** [matklad/xshell#83]
something like `Cmd::set_verbose(self, bool)` that if set will print to stderr `$ ENV1=KEY1 ENV2=KEY2 ... CMD ARGS` this will help debug errors when setting enviroment variables

**Comment by matklad** [comment]
Makes sense to me! We already have `Cmd::quiet / set_quiet` which controls echoing  of the command itself. One design question is the specific naming of methods. I think `fn quiet` is already wrong, as it is very unclear what that does. So perhaps we can deprecate that and come up with some clearer API

**Comment by elichai** [comment]
@matklad Maybe something like `echo_command()` and `echo_env()`?
